Formula/tere.rb

```diff
@@ -0,0 +1,36 @@
+class Tere < Formula
+  desc "Terminal file explorer"
+  homepage "https://github.com/mgunyho/tere"
+  url "https://github.com/mgunyho/tere/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "a4ebd2f0b5728d6ec61749564106d67d74720fa05cb906686d559a5f15549698"
+  license "EUPL-1.2"
+  head "https://github.com/mgunyho/tere.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  def caveats
+    <<~EOS
+      Shell wrappers must be installed for `tere` to work.
+      Follow the instructions listed here:
+        https://github.com/mgunyho/tere/blob/master/README.md#setup
+    EOS
+  end
```

Comment:
This is not specific to Homebrew so should not be here

Reply:
Got it. The section has been removed.